tools/shared/mach.c

```diff
@@ -76,7 +39,7 @@ set_mach(X86TYPE *mach, int machtype)
  *  -tp may not be appropriate for ARM; so machtype may refer to
  *  ARM attriubutes rather than a TP_ value.
  */
-#if !defined(TARGET_LLVM_ARM)  /* { */
+#if defined(TARGET_X86)  /* { */
```

Comment:
Heads-up: all targets that are not ARM != x86 targets.

Reply:
comment updated